### DISCUSSION TITLE
How to get pull shark badge?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#19507] [Accessibility] [answered]
Hey world! How do I get pull shark badge???

**Comment by D7EAD** [comment, marked as answer]
Hi!

You can get the pull shark badge by having your PRs merged into another repo! The pull shark badge has several tiers, all gained by an increasing number of merged PRs!

You can learn more about Github Achievements [here](https://github.com/Schweinepriester/github-profile-achievements).

**Reply by ghost** [reply]
Thanks onii!
